### PR TITLE
update sql-assessment to 0.5.0

### PR DESCRIPTION
**Pull request by vchvlad**

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -3200,8 +3200,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.4.0",
-							"lastUpdated": "11/5/2020",
+							"version": "0.5.0",
+							"lastUpdated": "12/1/2020",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:3204]: You missed updating the link below

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

**Comment by Charles-Gagnon** [comment]
And can you make sure that https://sqlopsextensions.blob.core.windows.net/extensions/sql-assessment/sql-assessment-0.5.0.vsix has all the fixes you expect to be there?

File: extensionsGallery.json (hunk 1 above)
Comment (vchvlad) [reply extensionsGallery.json:3204]: I've sent you link to the correct file vsix via email.

Comment (vchvlad) [reply extensionsGallery.json:3204]: Updated. Thanks.

**Comment by vchvlad** [comment]
> And can you make sure that https://sqlopsextensions.blob.core.windows.net/extensions/sql-assessment/sql-assessment-0.5.0.vsix has all the fixes you expect to be there?

Yes. Just checked.

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
@vchvlad Galleries updated

**Pull request merged**